scripts/planning/sprint-planning.sh

```diff
@@ -26,7 +26,7 @@ function usage()
 
 function github_milestone()
 {
-  milestone_data="`curl -s https://api.github.com/repos/minishift/$repository/issues?milestone=$milestone\&state=all`"
+  milestone_data="`curl -s https://api.github.com/repos/minishift/$repository/issues?per_page=100\&milestone=$milestone\&state=all`"
```

Comment:
Decided not to use the token as there is more than one condition to handle, if the token given is wrong, if the token is empty.